sound/soc/sof/ipc4.c

```diff
@@ -653,9 +653,24 @@ static int sof_ipc4_ctx_save(struct snd_sof_dev *sdev)
 	return sof_ipc4_set_core_state(sdev, SOF_DSP_PRIMARY_CORE, false);
 }
 
+static int sof_ipc4_set_pm_state(struct snd_sof_dev *sdev, u32 flags)
+{
+	struct sof_ipc4_msg msg;
+
+	memset(&msg, 0, sizeof(msg));
```

Comment:
struct sof_ipc4_msg msg = {{ 0 }}; instead? thats what we've done everywhere

Reply:
sure, thanks